scalableminds/webknossos
# Fix ignoring of reported datasets of unknown orga

This got lost in #4135 .

(the PR also removes the time logging)

### Steps to test:
- in your binaryData dir create a second directory, add a dataset into it
- hit “refresh” in wK
- requests should not fail, wk terminal output should say ignoring datasets…

------
